lib/content-services/src/lib/permission-manager/components/user-icon-column/user-icon-column.component.spec.ts

```diff
@@ -93,7 +95,8 @@ describe('UserIconColumnComponent', () => {
             component.ngOnInit();
             fixture.detectChanges();
             expect(getGroupIcon()).toBeDefined();
-            expect(getGroupIcon().textContent).toContain('people_alt_outline');
+            expect(getGroupIcon().nativeElement.textContent).toContain('people_alt_outline');
+            expect(testingUtils.getInnerTextByCSS('.cdk-visually-hidden')).toBe('USER_ICON.GROUP_ICON_ALT');
```

Comment:
.cdk-visually-hidden selector is used multiple times, can we wrap that in function?